cpp/modmesh/universe/bezier.hpp

```diff
@@ -1235,6 +1235,719 @@ using SegmentPadFp64 = SegmentPad<double>;
+    }
+    void set_at(size_t i, point_type const & p0, point_type const & p1, point_type const & p2)
+    {
+        x0_at(i) = p0.x();
+        y0_at(i) = p0.y();
+        x1_at(i) = p1.x();
+        y1_at(i) = p1.y();
+        x2_at(i) = p2.x();
+        y2_at(i) = p2.y();
+        if (ndim() == 3)
+        {
+            z0_at(i) = p0.z();
+            z1_at(i) = p1.z();
+            z2_at(i) = p2.z();
+        }
+    }
+    void set_at(size_t i, value_type x0, value_type y0, value_type x1, value_type y1, value_type x2, value_type y2)
+    {
+        x0_at(i) = x0;
+        y0_at(i) = y0;
+        x1_at(i) = x1;
+        y1_at(i) = y1;
+        x2_at(i) = x2;
+        y2_at(i) = y2;
+    }
+    void set_at(size_t i, value_type x0, value_type y0, value_type z0, value_type x1, value_type y1, value_type z1, value_type x2, value_type y2, value_type z2)
+    {
+        x0_at(i) = x0;
+        y0_at(i) = y0;
+        x1_at(i) = x1;
+        y1_at(i) = y1;
+        x2_at(i) = x2;
+        y2_at(i) = y2;
+        if (ndim() == 3)
+        {
+            z0_at(i) = z0;
+            z1_at(i) = z1;
+            z2_at(i) = z2;
+        }
+    }
+
+    triangle_type get(size_t i) const
+    {
+        if (ndim() == 3)
+        {
+            return triangle_type(point_type(x0(i), y0(i), z0(i)), point_type(x1(i), y1(i), z1(i)), point_type(x2(i), y2(i), z2(i)));
+        }
+        else
+        {
+            return triangle_type(point_type(x0(i), y0(i), 0.0), point_type(x1(i), y1(i), 0.0), point_type(x2(i), y2(i), 0.0));
+        }
+    }
+    void set(size_t i, triangle_type const & t)
+    {
+        x0(i) = t.x0();
+        y0(i) = t.y0();
+        x1(i) = t.x1();
+        y1(i) = t.y1();
+        x2(i) = t.x2();
+        y2(i) = t.y2();
+        if (ndim() == 3)
+        {
+            z0(i) = t.z0();
+            z1(i) = t.z1();
+            z2(i) = t.z2();
+        }
+    }
+    void set(size_t i, point_type const & p0, point_type const & p1, point_type const & p2)
+    {
+        x0(i) = p0.x();
+        y0(i) = p0.y();
+        x1(i) = p1.x();
+        y1(i) = p1.y();
+        x2(i) = p2.x();
+        y2(i) = p2.y();
+        if (ndim() == 3)
+        {
+            z0(i) = p0.z();
+            z1(i) = p1.z();
+            z2(i) = p2.z();
+        }
+    }
+    void set(size_t i, value_type x0, value_type y0, value_type x1, value_type y1, value_type x2, value_type y2)
+    {
+        x0(i) = x0;
+        y0(i) = y0;
+        x1(i) = x1;
+        y1(i) = y1;
+        x2(i) = x2;
+        y2(i) = y2;
+    }
+    void set(size_t i, value_type x0, value_type y0, value_type z0, value_type x1, value_type y1, value_type z1, value_type x2, value_type y2, value_type z2)
+    {
+        x0(i) = x0;
+        y0(i) = y0;
+        x1(i) = x1;
+        y1(i) = y1;
+        x2(i) = x2;
+        y2(i) = y2;
+        if (ndim() == 3)
+        {
+            z0(i) = z0;
+            z1(i) = z1;
+            z2(i) = z2;
+        }
+    }
+
+    void mirror_x()
```

Comment:
Leave comments to `mirror_*()` to clearly state that the functions are mirroring with respect to the axes.